app/lib/shared/integrity.dart

```diff
@@ -263,12 +263,14 @@ class IntegrityChecker {
     if (p.likes < 0) {
       yield 'Package "${p.name}" has a `likes` property which is not a non-negative integer.';
     }
-    for (final userId in p.uploaders!) {
-      yield* _checkUserValid(
-        userId,
-        entityType: 'Package',
-        entityId: p.name,
-      );
+    if (p.uploaders != null) {
+      for (final userId in p.uploaders!) {
```

Comment:
get some type promotion:
```suggestion
    final uploaders = p.uploaders;
    if (uploaders != null) {
      for (final userId in uploaders) {
```